css/styles.css

```diff
@@ -268,3 +268,71 @@ textarea {
   resize: vertical;
   height: 170px;
 }
+
+@media (min-width: 500px) {
+  body {
+    background-color: #172b4d;
+  }
+
+  .app-wrapper {
+    width: 100vw;
+    display: flex;
+    flex-direction: column;
+  }
+
+  .welcome-line {
+    background-image: url("/assets/shapes/background-shapes/desktop/desktop-header-shapes.svg");
+  }
```

Comment:
- [ ] Your implementation of the headline section is marvellous but the background image is not displayed on the website. This might be because of differences in the file name or filling improper file location in the `url` tag.

### Your implementation
![image](https://user-images.githubusercontent.com/59416136/154524824-b7800d3f-9721-40f4-a09b-41647a033c52.png)

### Figma's Design
![image](https://user-images.githubusercontent.com/59416136/154524919-7d9c7f2d-365e-4668-93a2-d75b84cc26de.png)

Kindly fix that.